commit/report.go

```diff
@@ -16,8 +16,8 @@ import (
 // ReportInfo is the info data that will be sent with the along with the report
 // It will be used to determine if the report should be accepted or not
 type ReportInfo struct {
-	// MinSigners is the minimum number of RMN signatures required for the report to be accepted
-	MinSigners uint64 `json:"minSigners"`
+	// RemoteF Max number of faulty RMN nodes; f+1 signers are required to verify a report.
+	RemoteF uint64 `json:"remoteF"`
```

Comment:
This F is different than the source chain F right? This is the F for report signatures and not observations?

Maybe for terminology we can call it `ReportF` and `ObservationF`? @gtklocker thoughts?